<launch>
  <node pkg="autoware_hazard_status_converter" exec="autoware_hazard_status_converter_node" name="hazard_status_converter">
    <param from="$(find-pkg-share autoware_hazard_status_converter)/config/hazard_status_converter.param.yaml"/>
    <remap from="~/diagnostics_graph" to="/diagnostics_graph"/>
    <remap from="~/hazard_status" to="/system/emergency/hazard_status"/>
    <remap from="~/input/emergency_holding" to="/system/emergency_holding"/>
  </node>
</launch>
